ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource

```diff
@@ -225,7 +227,7 @@ Get Model Serving Access Token via UI
         SeleniumLibrary.Wait Until Page Contains Element    xpath://td[@data-label="Tokens"]/button
         SeleniumLibrary.Click Element    xpath://td[@data-label="Tokens"]/button
         ${token}=    SeleniumLibrary.Get Element Attribute
-        ...    xpath://div[.="${service_account_name}"]/../../td[@data-label="Token Secret"]//span/input    value
+        ...    xpath://td[@data-label='Token Secret']//input   value
```

Comment:
we may have more than one service account

Reply:
yes Changed it